docs/operation/security.md

```diff
@@ -4,6 +4,28 @@ Security is a critical aspect of any application. In this section, we will discu
 
 ## Authentication
 
+### Username/Password Authentication
+
+BanyanDB supports username/password authentication for gRPC and HTTP server. The following flag is used to configure authentication:
+
+- `--auth-config-file`: Path to the authentication config file (YAML format).
+
+Password should be encrypted by bcrypt. For example, you can create an authentication configuration file like this:
+
+```yaml
+users:
+  - username: admin
+    password: $2a$10$Dty9D1PMVx0kt24S09qs6ezn2Q77wLsnmlpU6iO29hMn.Urbo.uji
```

Comment:
It would be helpful to provide tools that assist users in getting the hash string from their raw passwords.